CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+
+---
+
+## 🚩 Quick Start
+
+### 1. **Get Access**
+- Make sure you have a [GitHub account](https://github.com/).
+- Request access to this repository if needed.
+- Request access to Figma if needed. Please request view access. If edit access is needed, we can grant it, otherwise, we have a collective account, or you can duplicate files to your personal Figma account.
+
+### 2. **Clone the Repo**
+```bash
+git clone https://github.com/openfoodfacts/openfoodfacts-design.git
+cd openfoodfacts-design
+```
+
+### 3. **Explore the Directory**
+NOT CREATED YET, STRUCTURE TO DEBATE
+- `/assets`: Design files (Figma exports, SVGs, etc.)
+- `/components`: UI components and guidelines
+- `/docs`: Documentation (including this guide!)
+- `/ux-research`: Research plans, reports, and insights
+
+---
+
+## 🛠️ For Designers
+
+- **Design System:** Find the [Figma file](https://www.figma.com/) link in `/assets` or ask in Slack.
+- **Additions:** Propose changes via issues or PRs. Attach Figma/Sketch/other files as needed.
+- **Naming:** Use clear, descriptive names for assets and components.
+- **Accessibility:** Follow [WCAG guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) in your designs.
+
+---
+
+## 💻 For Developers
+
+- **Design Tokens:** Sync with `/components/tokens` for colors, spacing, typography, etc.
+- **Implementation:** Check `/components` for specs and code snippets.
+- **Feedback Loop:** Collaborate with designers early; raise issues if specs are unclear or incomplete.
+- **Storybook (if available):** Run Storybook locally to view UI components in isolation.
+
+---
+
+## 🔬 For UX Researchers
+
+- **Research Directory:** Use `/ux-research` for plans, surveys, and reports.
+- **Issue Tracking:** Log findings as GitHub issues and tag relevant teams.
+- **Collaboration:** Work with designers/devs to turn research into actionable design changes.
+
+---
+
+## 📣 Communication
+
+- **Slack:** Join our [#design channel](https://openfoodfacts.slack.com/) for quick questions and discussions.
+- **GitHub Issues:** Use issues for bugs, feature requests, or research tasks.
+- **Meetings:** Participate in regular syncs (calendar invites sent via Slack).
+
+## Weekly meetings (Design team only)
+
+- We e-meet on Fridays at 09:00 Paris Time (08:00 London Time, 11:30 IST, 12:00 AM PT)
+- ![Google Meet](https://img.shields.io/badge/Google%20Meet-00897B?logo=google-meet&logoColor=white) Video call link: <https://meet.google.com/gnp-frks-esc>
+- Join by phone: <https://tel.meet/gnp-frks-esc?pin=1110549945262>
+- Add the Event to your Calendar by [adding the Open Food Facts community calendar to your calendar](https://wiki.openfoodfacts.org/Events)
+- [Weekly Agenda](https://docs.google.com/document/d/1MGQqMV7M4JTjFcRsiRvMZ8bnmd9vJWdSyRR3wJHUBMk/edit): please add the Agenda items as early as you can. Make sure to check the Agenda items in advance of the meeting, so that we have the most informed discussions possible, leading to argumented decisions.
```

Comment:
```suggestion
- [Weekly Agenda](https://docs.google.com/xxxxx/edit): please add the Agenda items as early as you can. Make sure to check the Agenda items in advance of the meeting, so that we have the most informed discussions possible, leading to argumented decisions.
```